pkg/fieldpath/fieldpath.go

```diff
@@ -123,6 +123,31 @@ func ExtractContainerResourceValue(fs *v1.ResourceFieldSelector, container *v1.C
 	return "", fmt.Errorf("Unsupported container resource : %v", fs.Resource)
 }
 
+// TODO: remove this duplicate
+// InternalExtractContainerResourceValue extracts the value of a resource
+// in an already known container
+func InternalExtractContainerResourceValue(fs *api.ResourceFieldSelector, container *api.Container) (string, error) {
```

Comment:
The ExtractContainerResourceValue method shouldn't even exist in this package - really surprised it got cloned into here.